src/foam/blob/Blob.js

```diff
@@ -477,7 +477,8 @@ foam.CLASS({
   properties: [
     {
       class: 'String',
-      name: 'root'
+      name: 'root',
```

Comment:
Is this property still used? Can you document what it's for?

Reply:
Yes it is, yes I can